ci/do_ci.sh

```diff
@@ -83,8 +110,8 @@ elif [[ "$1" == "bazel.legacy.test" ]]; then
   bazel test $BAZEL_TEST_OPTIONS -- //... -//exporters/otlp/...
   exit 0
 elif [[ "$1" == "bazel.noexcept" ]]; then
-  bazel build --copt=-fno-exceptions $BAZEL_OPTIONS //...
-  bazel test --copt=-fno-exceptions $BAZEL_TEST_OPTIONS //...
+  bazel build --copt=-fno-exceptions $BAZEL_OPTIONS -- //... -//exporters/prometheus/...
```

Comment:
Why would we want to take exception here?

Reply:
When we run the test, we found some error handling and exceptions from the Prometheus client, which made this test always fail. We discussed this problem with Johannes last week, and he suggested us to consider excluding the Prometheus component in the noexcept test.

Reply:
Please add a comment here.